src/Analyzers/CSharp/Tests/UseAutoProperty/UseAutoPropertyTests.cs

```diff
@@ -684,11 +686,11 @@ int P
                     }
                 }
             }
-            """);
+            """, new TestParameters(parseOptions: CSharp12));
```

Comment:
not supported in c# 12.  but will work in c# 13.  tests for those are in teh sibling file added.